code/Items/Slot.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Sandbox;
+using ssl.Items.Data;
+
+namespace ssl.Items
+{
+    public class Slot : NetworkComponent
+    {
+        public ItemStack ItemStack { get; set; }
```

Comment:
Needs to be networked, the class Slot should extend from our workaround NetworkAllTransmitted